lib/components/viewers/realtime-status-label.tsx

```diff
@@ -108,19 +118,27 @@ const RealtimeStatusLabel = ({
       {renderedTime}
       <MainContent>
         <FormattedRealtimeStatusLabel
-          minutes={isEarlyOrLate && (
-            <DelayText>
-              <FormattedDuration duration={Math.abs(delay)} />
-            </DelayText>
-          )}
+          minutes={
+            isEarlyOrLate ? (
+              <DelayText>
+                <FormattedDuration duration={Math.abs(delay)} />
+              </DelayText>
+            ) : (
+              <>{null}</>
```

Comment:
Just `null` should be enough, no?

Reply:
If only... Typescript was very upset about the JSX being passed in as a value for FormattedMessage so I had to wrap the null to make it happy